terraform/cloudfoundry/dns-records.tf

```diff
@@ -0,0 +1,7 @@
+resource "aws_route53_record" "wildcard" {
+  zone_id = "${var.dns_zone_id}"
+  name = "*.${var.env}.${var.dns_zone_name}."
+  type = "CNAME"
+  ttl = "60"
+  records = ["dummy"]
```

Comment:
Why we use the value "dummy" here, instead of using the public IP  of the web ELB: `${aws_elb.router.dns_name}`, as done [here](https://github.com/alphagov/cf-terraform/blob/master/aws/dns-records.tf#L6)?